assets/constants.ts

```diff
@@ -4876,6 +4876,9 @@ export = {
   }, {
     "id": "fr",
     "text": "français (French)"
+  }, {
+    "id": "kab",
```

Comment:
Let's drop this, it looks like it's already in the list below. (But maybe change that entry to add "(Kabyle)" in parens?)

Reply:
Oh, yes!  I'll remove it.

Reply:
Done!